doc/source/debian.rst

```diff
@@ -1,48 +1,54 @@
-Create a Debian Squeeze jail (GNU/kFreeBSD)
+.. index:: Create Debian Squeeze Jail
+.. _Create a Debian Squeeze Jail:
+
+Create a Debian Squeeze Jail (GNU/kFreeBSD)
 ===========================================
 
-**In this howto we will set up a Debian (GNU/kFreeBSD) jail. GNrUkFreeBSD is a
-Debian userland tailored for the FreeBSD kernel.**
+This section shows the process to set up a Debian (GNU/kFreeBSD) jail.
+GNrUkFreeBSD is a Debian userland tailored for the FreeBSD kernel.
+
+The examples in this section use a jail with the custom tag **debjail**.
+Remember to replace **debjail** with your jail's UUID or TAG!
+
+.. warning:: This is not recommended for production use. The intention
+   is to show :command:`iocage` can do almost anything with jails.
 
-Don't forget to replace UUID with your jail's full UUID!
+**Create an empty jail with Linux specifics:**
 
-**Create an empty jail with linux specifics:**
+.. code-block:: none
 
-``iocage create -e tag=debian exec_start="/etc/init.d/rc 3"
-exec_stop="/etc/init.d/rc 0"``
+ # iocage create -e tag=debjail exec_start="/etc/init.d/rc 3" exec_stop="/etc/init.d/rc 0"
 
 **Install debootstrap on the host:**
 
-``pkg install debootstrap``
+:samp:`# pkg install debootstrap`
 
-**Grab the mountpoint for our empty jail, append /root/ to it and run
-debootstrap:**
+**Grab the mountpoint for the empty jail, append /root/ to it, and run**
+**debootstrap:**
 
-``iocage get mountpoint UUID | TAG``
+:samp:`# iocage get mountpoint debjail`
 
-``debootstrap squeeze /iocage/jails/UUID/root/`` (you can replace squeeze with wheezy if that is what you need)
+:samp:`# debootstrap squeeze /iocage/jails/debjail/root/`
 
-**Edit the jail's fstab and add these lines:**
+Replace *squeeze* with *wheezy*, if needed.
 
-``/iocage/jails/UUID/fstab``
+**Edit the jail** :file:`fstab` **and add these lines:**
```

Comment:
`iocage fstab -e debjail` or we can show them how to do it with a call for each mount to `iocage fstab -a debjail FSTAB`